krabs/krabs/filtering/comparers.hpp

```diff
@@ -30,7 +30,7 @@ namespace krabs { namespace predicates {
         struct equals
         {
             template <typename Iter1, typename Iter2>
-            bool operator()(Iter1 begin1, Iter1 end1, Iter2 begin2, Iter2 end2)
+            bool operator()(Iter1 begin1, Iter1 end1, Iter2 begin2, Iter2 end2) const
```

Comment:
These are required to make the `property_*` predicates match the other predicates like `id_is`, etc.